.vscode/settings.json

```diff
@@ -1,5 +0,0 @@
-{
-    "cSpell.words": [
-        "grayscale"
-    ]
-}
```

Comment:
Why are we deleting this file?

Reply:
Oops, that was an accident.

Reply:
I went ahead and restored it (I would have asked but I didn't realize the "restore file" button would just do it rather than asking me).